awx/playbooks/project_update.yml

```diff
@@ -232,20 +233,37 @@
     - name: Fetch galaxy collections from collections/requirements.(yml/yaml)
       ansible.builtin.command: >
         ansible-galaxy collection install -r {{ item }}
-        --collections-path {{ projects_root }}/.__awx_cache/{{ local_path }}/stage/requirements_collections
```

Comment:
So I'm hoping that this was already unnecessary because `additional_collections_env` set the same thing via `ANSIBLE_COLLECTIONS_PATHS`